### PR TITLE
Adding keyboard S20 by polycarbdiet

### PR DESCRIPTION
**Pull request by polycarbdiet**
## Description

Addition of a new keyboard - S20. This is an FR4 sandwich keyboard that I designed for my use but am planning on sharing with others, including a GB.

My previous PR had too many commits that weren't needed. I am resubmitting this PR.  
Please let me know if this isn't the right way to go about things.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/polycarbdiet/s20/rules.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = ortho_5x4 numpad_5x4 enter
```
Comment (zvecr) [inline keyboards/polycarbdiet/s20/rules.mk:23]:
```suggestion
LAYOUTS = ortho_5x4 numpad_5x4
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by polycarbdiet** [comment]
@zvecr : Hello! I was wondering why we don't include this in the rules.mk as one of the supported layouts?

**Comment by zvecr** [comment]
LAYOUTS marks supported community layouts not everything this keyboard has. See https://docs.qmk.fm/#/feature_layouts for more information on the topic.

**Comment by polycarbdiet** [comment]
> LAYOUTS marks supported community layouts not everything this keyboard has. See https://docs.qmk.fm/#/feature_layouts for more information on the topic.

I have just viewed this page, thanks for showing me the info!

I have made your recommended changes. Is there anything else that I need to do? Apologies, I am still new to this process.

**Comment by polycarbdiet** [comment]
I'll resubmit this PR due to unnecessary commits. Thanks for the help!

**Pull request closed without merging**